packages/typo3-docs-theme/assets/sass/components/_code.scss

```diff
@@ -90,63 +93,8 @@
  */
 .code-inline {
     font-family: $font-family-monospace;
-    font-size: 90%;
     border-radius: $border-radius;
     border: 1px solid #fff;
-
-    &.bash {
```

Comment:
Then we should switch to use `:code:` generally and also recommend it on migration.

Reply:
Additionally, the node.class should not be necessary anymore?
https://github.com/TYPO3-Documentation/render-guides/blob/main/packages/typo3-docs-theme/resources/template/inline/textroles/code.html.twig

Reply:
IMO the whole "icon design" is up for discussion at a later place, so that we may use hover or css :after(...) features to add "PHP", "YAML" etc. to some kind of box. IMO we can remove the coloring at this point because it needs its own treatment, and currently the colors are really detracting from reading....?

Reply:
Then we should tackle this beforehand. Otherwise we encourage people to use `code` as there is no difference visible.